text/0016-named-tracers.md

```diff
@@ -14,41 +14,49 @@ _Creating Tracers and Meters using a factory mechanism and naming those Tracers
 
 The mechanism of "Named Tracers and Meters" proposed here is motivated by following scenarios:
 
-* For a consumer of OpenTelemetry instrumentation libraries, there is currently no possibility of influencing the amount of the data produced by such libraries. Instrumentation libraries can easily "spam" backend systems, deliver bogus data or - in the worst case - crash or slow down applications. These problems might even occur suddenly in production environments caused by external factors such as increasing load or unexpected input data.
+**Expensive instrumentation**
```

Comment:
The paragraph below not only covers excessive instrumentation but also buggy ones which deliver incorrect data. What about "Faulty or excessive instrumentation"?